app/src/main/java/com/x8bit/bitwarden/data/vault/repository/VaultRepositoryImpl.kt

```diff
@@ -523,6 +524,20 @@ class VaultRepositoryImpl(
             )
     }
 
+    override suspend fun getDecryptedFido2CredentialAutofillViews(
+        cipherViewList: List<CipherView>,
+    ): List<Fido2CredentialAutofillView> {
+        return vaultSdkSource
+            .decryptFido2CredentialAutofillViews(
+                userId = activeUserId ?: return emptyList(),
+                cipherViews = cipherViewList.toTypedArray(),
+            )
+            .fold(
+                onFailure = { emptyList() },
```

Comment:
So we do not want to report a failure here and always return a list?

Usually we would create a sealed class for this where only the success has the list of data.

Reply:
Ok, I can do that!